### PR TITLE
Typos in docs

### PR DESCRIPTION
**Pull request by rcooke-ast**
Might as well include these in the soon to be released version, too...

### CODE REVIEW

File: doc/calibrations/image_proc.rst [hunk 1]
```diff
@@ -140,7 +140,7 @@ Read and Digitization Noise
 
 Readnoise variance, :math:`V_{\rm rn}`, is calculated by
 :func:`~pypeit.core.procimg.rn2_frame`.  The calculation requires the detector
-readnoise (RN in elections, e-) and, possibly, gain (:math:`g` in e-/ADU)
+readnoise (RN in electrons, e-) and, possibly, gain (:math:`g` in e-/ADU)
```
Comment (kbwestfall) [inline doc/calibrations/image_proc.rst:143]: Ha!

**Review by kbwestfall — APPROVED** [review]
Must have had "elections" on the brain :). Thanks for these corrections!

**Comment by rcooke-ast** [comment]
> Must have had "elections" on the brain :). Thanks for these corrections!

Haha... I should also say thanks for writing these exhaustive docs... tremendously helpful when I was trying to understand why the dark current was implemented the way it is (in the context of KCWI).

**Review by debora-pe — APPROVED** [review]
Thanks!

**Pull request merged**